website/meta/universe.json

```
@@ -2599,6 +2599,53 @@
             },
             "category": ["pipeline"]
         },
+        {
+            "id": "classyclassification",
+            "slogan": "A python library for classy few-shot and zero-shot classification within Spacy.",
+            "description": "Huggingface does offer some nice models for few/zero-shot classification, but these are not tailored to multi-lingual approaches. Rasa NLU has a nice approach for this, but its too embedded in their codebase for easy usage outside of Rasa/chatbots. Additionally, it made sense to integrate sentence-transformers and Hugginface zero-shot, instead of default word embeddings. Finally, I decided to integrate with Spacy, since training a custom Spacy TextCategorizer seems like a lot of hassle if you want something quick and dirty.",
+            "github": "davidberenstein1957/classy-classification",
+            "pip": "classy-classification",
+            "code_example": [
+                "import spacy",
+                "import classy_classification",
+                "",
+                "data = {",
+                "    \"furniture\": [\"This text is about chairs.\",",
+                "               \"Couches, benches and televisions.\",",
+                "               \"I really need to get a new sofa.\"],",
+                "    \"kitchen\": [\"There also exist things like fridges.\",",
+                "                \"I hope to be getting a new stove today.\",",
+                "                \"Do you also have some ovens.\"]",
+                "}",
+                "",
+                "nlp = spacy.load('en_core_web_md')",
+                "",
+                "# few-shot using spacy internal embeddings via md or lg model",
+                "nlp.add_pipe(\"text_categorizer\", ",
+                "    config={\"data\": data, \"model\": \"spacy\"}",
+                ")",
+                "",
+                "# few-shot using sentence-transformers",
+                "nlp.add_pipe(\"text_categorizer\", ",
+                "    config={\"data\": data, \"model\": \"sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2\"}",
+                ")",
+                "",
+                "# zero-shot using huggingface",
+                "nlp.add_pipe(\"text_categorizer\", ",
+                "    config={\"data\": list(data.keys()), \"cat_type\": \"zero\", \"model\": \"facebook/bart-large-mnli\"}",
+                ")",
+                "",
+                "nlp(\"I am looking for kitchen appliances.\")._.cats",
```

This won't do anything when run in an IDE, so I'd suggest changing this to 
```suggestion
                "print(nlp(\"I am looking for kitchen appliances.\")._.cats)",
```